rbac/management/principal/proxy.py

```diff
@@ -164,10 +170,16 @@ def request_principals(self, account, limit=None, offset=None):
                                       self.port,
                                       self.path,
                                       account_principals_path)
-        return self._request_principals(url, params=params)
+
+        # For v1 account users endpoints are already filtered by account
+        return self._request_principals(url, params=params, account_filter=False)
 
     def request_filtered_principals(self, principals, account=None, limit=None, offset=None):
         """Request specific principals for an account."""
+        if account is None:
```

Comment:
I was going to propose that we just use presence of "account" and get rid of  "account_filter", but I think we should leave things as they are as a way to document the behavior.

Reply:
I see!  That should be a pretty simple change if we want to make it after we test it out!